e2e/utils/add-growthbook-cookie.ts

```diff
@@ -11,7 +11,9 @@ export async function addGrowthbookCookie({
     {
       name: 'gbuuid',
       value: variation,
-      domain: 'localhost',
+      // Weirdly, it seems necessary to prefix the domain with a dot. This seems
+      // to be a peculiarity of Playwright.
+      domain: process.env.COOKIE_DOMAIN || 'localhost',
```

Comment:
I do not see a _dot_?

Reply:
```suggestion
      // Weirdly, it seems necessary to prefix the domain in COOKIE_DOMAIN i.e.
      // if the domain is freecodecamp.org then the COOKIE_DOMAIN must be
      // .freecodecamp.org. This seems to be a peculiarity of Playwright.
      domain: process.env.COOKIE_DOMAIN || 'localhost',
```
Yeah, that could have been clearer. How about this?

Reply:
Does that mean this needs to change?

https://github.com/freeCodeCamp/freeCodeCamp/blob/43661cf867796037f93f1a7fb640f94b2cd1d1ea/.gitpod.yml#L33

Reply:
Good catch. I'll try changing that and see if anything breaks (it shouldn't).

Reply:
Seems fine.